src/components/NotificationPopup.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <ListSection class="notification-popup" v-if="visible">
+    <Li
+      v-for="notification in notificationList"
+      :key="notification.id"
+      :class="notification.level"
+      :listEntry="notification"
+    />
+  </ListSection>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop } from 'vue-property-decorator';
+import ListSection from '@/components/ListSection.vue';
+import Li, { ListEntry } from '@/components/Li.vue';
+import Utils from '@/utils/Utils';
+
+@Component({
+  components: {
+    ListSection,
+    Li,
+  },
+})
+export default class NotificationPopup extends Vue {
+  // region public static methods
+  // endregion
+
+  // region private static methods
+  // endregion
+
+  // region public members
+  @Prop(Boolean)
+  public visible!: boolean;
+
+  private lastUpdate: number = Number.MAX_SAFE_INTEGER;
+  private timer: number = 0;
+
+  get notificationList(): ListEntry[] {
+    const newNotifications = this.$notifications.notifications.filter(
+      (notification) => {
+        if (!notification.timestamp) {
```

Comment:
https://github.com/bptlab/rembrandt-frontend/blob/cd30cdb3a1847d5816675c2292ecedbb81d041e3/src/plugins/Notification.ts#L50
ensures that you always have a timestamp. And the notification center also assumes that.
So this `if` might be removed.

Reply:
done